File: springboot-reactjs-using-template/src/main/java/app/CoffeeShopRepository.java
package app;

import org.springframework.data.repository.CrudRepository;
import org.springframework.data.rest.core.annotation.RepositoryRestResource;

@RepositoryRestResource(collectionResourceRel = "coffeeshops", path = "coffeeshops")
public interface CoffeeShopRepository extends CrudRepository<CoffeeShopModel, Long> {
}

File: springboot-kotlin-reactjs/src/main/kotlin/org/example/CoffeeShopRepository.kt
package org.example

import org.springframework.data.repository.CrudRepository
import org.springframework.data.rest.core.annotation.RepositoryRestResource

@RepositoryRestResource(collectionResourceRel = "coffeeshops", path = "coffeeshops")
interface CoffeeShopRepository : CrudRepository<CoffeeShopModel, Long> {
}